recipes-core/images/openvario-base-image.bb

```diff
@@ -42,6 +42,7 @@ IMAGE_INSTALL = " \
     nano \
     openssh-sftp-server \
     pulseaudio-server \
+    git \
```

Comment:
I'd recommend to add it to `DEPENDS` variable above instead adding it to image itself. This way it is installable with `opkg` for those who need it, but does not add to image size.

Reply:
Agreed, done.